.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
 	"extends": "@guardian/eslint-config",
-	"ignorePatterns": ["dist"],
+	"ignorePatterns": ["dist", "script"],
```

Comment:
I did this because I was getting a weird eslint error about a bracket in the `script/test.js` file but maybe there's a better fix?

Reply:
Hoping this is fixed in #39